Task_functions/index.js

```diff
@@ -1,11 +1,15 @@
 //Write code for the 1st task below this
```

Comment:
The function name 'addTWo' should be 'addTwo' to follow standard naming conventions.